search-service/src/test/kotlin/com/egm/stellio/search/temporal/service/AggregatedTemporalQueryServiceTests.kt

```diff
@@ -39,10 +43,18 @@ class AggregatedTemporalQueryServiceTests : WithTimescaleContainer, WithKafkaCon
     @Autowired
     private lateinit var r2dbcEntityTemplate: R2dbcEntityTemplate
 
+    @MockkBean(relaxed = true)
+    private lateinit var searchProperties: SearchProperties
+
     private val now = ngsiLdDateTime()
     private val attributeUuid = UUID.randomUUID()
     private val entityId = "urn:ngsi-ld:BeeHive:${UUID.randomUUID()}".toUri()
 
+    @BeforeEach
```

Comment:
have you tried using @TestPropertySource(properties = [""])

Reply:
it will apply to all my tests and I need a special config for one test in the class